engine/lite/request-response/components-api/src/main/scala/pl/touk/nussknacker/engine/requestresponse/api/RequestResponseSourceFactory.scala

```diff
@@ -31,6 +32,11 @@ trait RequestResponseSource[T] extends BaseLiteSource[Any] {
   def openApiDefinition: Option[OpenApiSourceDefinition] = None
 
   override def transform(record: Any): Context = {
-    new BasicContextInitializingFunction[Any](contextIdGenerator, VariableConstants.InputVariableName)(record)
+    record match {
```

Comment:
Let's pass to RR sources additional http request fields like headers instead. Thanks to that, contextId can be extracted from headers